### PR TITLE
Keyboard shortcut to delete frame (fixes #5369)

### LINKED ISSUE #5369: Keyboard shortcut to delete frame
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

When we annotate videos, there are usually a lot of duplicated frames where we need to delete frames aggressively. The current delete frame function only support mouse click & involve a message prompt, which can be too slow for some users.

### Expected Behaviour
A keyboard shortcut that can quickly delete a frame by pressing a single key, and without prompting for confirmation.

### Current Behaviour
Require a mouse click and pop up a confirmation prompt.

### PR DESCRIPTION
**Pull request by scottykwok**
### Motivation and context
This fixes #5369 by adding keyboard shortcut [x] that delete a frame without prompt.
image

### How has this been tested?

Rebuild cvat/ui docker image locally and tested locally.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by nmanovic** [comment]
@scottykwok , thanks for the contribution! Let's see if all tests are passed. Could you please add a function test for the feature? Thus we will be sure that it works after our modifications.

@klakhov , could you please review the PR?

**Comment by bsekachev** [comment]
I am not sure deleting frame by ``X`` is a good idea. Too many functional keys near the ``X``. It might be often pressed accidentaly. 
Also, I believe, deleting a frame should work with the same key ``Del`` as deleting in object (maybe with a modificator, etc. Need to think more about it.)

**Comment by klakhov** [comment]
I'd agree with @bsekachev. It would be much better with `Alt+Del`.

**Comment by scottykwok** [comment]
Agreed. Alt+Del is a good suggestion. 
Then I will try update the pull request accordingly.

**Comment by nmanovic** [comment]
@scottykwok , please let us know if you want to improve the PR?

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ from online detectors & interactors) (<https://github.com/opencv/cvat/pull/4543>
 - Authentication with social accounts google & github (<https://github.com/opencv/cvat/pull/5147>, <https://github.com/opencv/cvat/pull/5181>, <https://github.com/opencv/cvat/pull/5295>)
 - REST API tests to export job datasets & annotations and validate their structure  (<https://github.com/opencv/cvat/pull/5160>)
 - Propagation backward on UI (<https://github.com/opencv/cvat/pull/5355>)
+- Keyboard shortcut to delete frame (<https://github.com/opencv/cvat/pull/5369>)
```
Comment (bsekachev) [inline CHANGELOG.md:24]:
```suggestion
- Keyboard shortcut to delete a frame (Alt + Del) (<https://github.com/opencv/cvat/pull/5369>)
```

**Review by bsekachev — APPROVED** [review]
LGTM, Thank you for the contribution!

**Pull request merged**